applications/radisys/readme.md

```diff
@@ -0,0 +1,80 @@
+**Radisys Engage Media Server**
+
+Radisys Corporation ©
+
+Engage Media Server is a media processing powerhouse for a wide range of
+revenue-generating services. Within a single platform, Radisys enables
+voice and video processing for real-time communication applications,
+audio and video mixing, transcoding, speech and video analytics, and
+other advanced media processing capabilities. It enables exciting
+solution offerings in UC and collaboration, audio and video
+conferencing, conversational AI-based speech-enabled applications,
+augmented reality, computer vision, multi-access edge media
+applications, VoLTE, ViLTE, and VoNR, transcoding, and more.
+
+The Engage Media Server portfolio supports rich conferencing
+capabilities through a variety of open programming interfaces, including
+Media Server Markup Language (MSML). Through these interfaces,
+conferencing application developers can program and dynamically control
+Engage Media Server's extensive real-time collaboration capabilities,
+voice and video quality optimizations, and switched and continuous
+presence modes for HD video conferencing. For mobile conferencing
+services, Engage Media Server media server supports IR.92 audio and
+IR.94 video compliant conferencing capabilities. Support for wide
+variety of HD audio and video codecs allows service providers to offer
+ubiquitous multimedia conferencing experiences with automatic
+transcoding across TDM, VoIP, VoLTE, VoNR, VoWiFi, and WebRTC endpoints.
+
+  **Virtual Platforms**               **Minimum**
+  ----------------------------------- --------------
+  Classification (Filtered Port 80)   NA
+  Delivery                            NA
+  Application                         Media server
+  Compute\* (vCores)                  4
+  Memory (RAM)                        4 GB
+  Storage (System minimum)            30 GB
+  Storage (Content maximum)           
+  Storage Performance                 
+  Host OS                             CentOS 7.x
+  Network                             1G / 10G
+
+**How to Install the vMRF Application**
+
+helm install vmrf \<helmPath\>
+
+## Test Output
+
+NAME: vmrf\
+LAST DEPLOYED: Thu Oct 29 23:44:38 2020\
+NAMESPACE: default\
+STATUS: pending-install
+
+## Uninstall vMRF Application
+
+helm uninstall vmrf
+
+## Test Output
+
+\# helm uninstall vmrf\
+release \"vmrf\" uninstalled
+
+# **Where to Purchase**
+
+Contact
+
+[Radisys Sales Support](https://go.radisys.com/vMRFSolutionlp.html) or
+<https://go.radisys.com/vMRFSolutionlp.html>
+
+Or write mail to sales\@radisys.com
+
+Alternatively, you can call Sales Support: (503) 615-1240
+
+**Resources**
+
+https://hub.radisys.com/mediaenginestream/rsys-computer-vision-whitepaper-101019-c?utm_campaign=cvwp&utm_medium=intelnetworkbuilders
```

Comment:
To show as bullet points:
~~~
* https://hub.radisys.com/mediaenginestream/rsys-computer-vision-whitepaper-101019-c?utm_campaign=cvwp&utm_medium=intelnetworkbuilders
* https://hub.radisys.com/mediaenginestream/biometric-authentication-solution-brief?utm_campaign=biometricauthentication&utm_medium=intelnetworkbuilders
* https://hub.radisys.com/mediaenginestream/ds-mediasrvr?utm_campaign=vmrfdatasheet&utm_medium=intelnetworkbuilders
* https://radisys.com
~~~